STYLE.md

```diff
@@ -109,6 +109,7 @@ spelled the same.
 | Use | Don't use | Notes |
 |-|-|-|
 | adaptor signatures | adapter signatures | |
+| discreet log contracts | discrete<!-- --> log contracts | Remembering to spell this correctly more often than one time in ten is known as the discrete log problem |
```

Comment:
I don't understand the `<!-- -->` here. Is that an html comment?

Reply:
Yes, an empty comment so that the `git grep` used in the test doesn't match this deliberate violation of the rule.  For the same reason, we use `[d]` instead of `d` in the actual pattern in the Makefile.

Reply:
ah! Thanks